include/swift/IDE/Formatting.h

```diff
@@ -20,8 +20,8 @@ struct CodeFormatOptions {
 public:
   bool UseTabs = false;
   bool IndentSwitchCase = false;
-  unsigned IndentWidth = 4;
-  unsigned TabWidth = 4;
+  unsigned IndentWidth = 2;
+  unsigned TabWidth = 8;
```

Comment:
Since you asked…please leave these alone. I like your settings better, but the current defaults match what Xcode does by default.